install/kubernetes/helm/subcharts/gateways/templates/deployment.yaml

```diff
@@ -152,6 +165,12 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
+          {{- if eq $key "istio-ingressgateway" }}
+          {{- if $spec.ingressSds.enabled }}
+          - name: ISTIO_META_INGRESS_SDS
```

Comment:
How about ISTIO_META_SDS ? To indicate that this node supports SDS. 

INGRESS doesn't have any meaning - Pilot already know the object is a router ( == gateway ).

Reply:
I would prefer using ISTIO_META_INGRESSS_SDS.
Gateway has two SDS clients: one for the external key/cert and one for the internal MTLS key/cert.

Reply:
I agree with @myidpt , use same uds socket will cause unpredictable result once nodeagent on the vm listens on the same socket as ingressgateway agent now.

Reply:
is this for user provided certs? Then please call it ISTIO_META_USER_SDS while the other one is ISTIO_META_MTLS_SDS